docs/source/rbac/scopes.md

```diff
@@ -0,0 +1,37 @@
+# Scopes
+
+A scope has a syntax-based design that reveals which resources it provides access to:
```

Comment:
In this section I think it'd be helpful to use different syntax to separate strings which are valid in scopes (e.g. `users`, `admin`) from placeholders like `resource`. E.g. `admin:<resource>`?